.github/workflows/build-on-release.yml

```diff
@@ -21,6 +21,10 @@ jobs:
             sm_branch: "1.11-dev"
             spcomp_version: "1.11.x"
 
+          - sm_version: "1.12"
+            sm_branch: "1.12-dev"
+            spcomp_version: "1.12.x"
```

Comment:
If I recall correctly spcomp 1.12 has flawed codegen due to the major refactoring work done on sourcepawn's side, so I'd probably opt to keep this at 1.11.x.
SourceMod plugin files are generally architecture-independent and there currently aren't any changes in plugin-space that improve support of 64-bit memory.